docs/plugins/inputs/kafka.asciidoc

```diff
@@ -21,26 +21,11 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-This input will read events from a Kafka topic. It uses the 0.10 version of
-the consumer API provided by Kafka to read messages from the broker.
-
-Here's a compatibility matrix that shows the Kafka client versions that are compatible with each combination
-of Logstash and the Kafka input plugin: 
-
-[options="header"]
-|==========================================================
-|Kafka Client Version |Logstash Version |Plugin Version |Why?
-|0.8       |2.0.0 - 2.x.x   |<3.0.0 |Legacy, 0.8 is still popular 
-|0.9       |2.0.0 - 2.3.x   | 3.x.x |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
-|0.9       |2.4.x - 5.x.x   | 4.x.x |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
-|0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |Not compatible with the <= 0.9 broker
-|0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |
-|0.11.0.0  |2.4.x - 5.x.x   | 6.x.x |Not compatible with the <= 0.9 broker
-|==========================================================
-
-NOTE: We recommended that you use matching Kafka client and broker versions. During upgrades, you should
-upgrade brokers before clients because brokers target backwards compatibility. For example, the 0.9 broker
-is compatible with both the 0.8 consumer and 0.9 consumer APIs, but not the other way around.
+This input will read events from a Kafka topic. 
+
+This plugin uses Kafka Client 0.11.0.0. For broker compatibility, see the official https://cwiki.apache.org/confluence/display/KAFKA/Compatibility+Matrix[Kafka compatibility reference].
```

Comment:
the [.lock file for the 6.1 branch](https://github.com/elastic/logstash/blob/6.1/Gemfile.jruby-2.3.lock.release#L326) mentions  [logstash-input-kafka (8.0.4) which depends on kafka 1.0.0](https://github.com/logstash-plugins/logstash-input-kafka/blob/v8.0.4/logstash-input-kafka.gemspec#L21)

Reply:
Well, the last release (6.1.1) uses 0.11.0.0:

```
% find ./ -name '*kafka-clients*jar'
./vendor/bundle/jruby/2.3.0/gems/logstash-input-kafka-8.0.2/lib/org/apache/kafka/kafka-clients/0.11.0.0/kafka-clients-0.11.0.0.jar
./vendor/bundle/jruby/2.3.0/gems/logstash-input-kafka-8.0.2/vendor/jar-dependencies/runtime-jars/kafka-clients-0.11.0.0.jar
./vendor/bundle/jruby/2.3.0/gems/logstash-output-kafka-7.0.4/lib/org/apache/kafka/kafka-clients/0.11.0.0/kafka-clients-0.11.0.0.jar
./vendor/bundle/jruby/2.3.0/gems/logstash-output-kafka-7.0.4/vendor/jar-dependencies/runtime-jars/kafka-clients-0.11.0.0.jar
```

Reply:
I guess 0.11.0.0 is the correct answer right now, so LGTM